packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -161,7 +165,26 @@ const Dashboard = () => {
                 isSaving={isSaving}
                 hasTilesChanged={hasTilesChanged}
                 onAddTile={onAddTile}
-                onSaveDashboard={() => mutate({ tiles: dashboardTiles })}
+                onSaveDashboard={() =>
+                    mutate({
+                        tiles: dashboardTiles,
+                        filters: {
```

Comment:
So this needs to transform `{ dashboardFilters } = useDashboard()` ?

Reply:
We haven't worked on the FE yet, just added this to make lint pass. 
This component will need a refactor because the DashboardProvider in declared here, so we can't use that hook in here as well. cc @nathaliabj